packages/components/select-v2/src/select.vue

```diff
@@ -226,12 +226,17 @@
           <template #default="scope">
             <slot v-bind="scope" />
           </template>
-          <template #empty>
-            <slot name="empty">
-              <p :class="nsSelect.be('dropdown', 'empty')">
-                {{ emptyText ? emptyText : '' }}
-              </p>
-            </slot>
+          <template v-if="$slots.loading && loading" #loading>
```

Comment:
IIRC. `#loading` here is actually redundant because the following `<slot name="loading" />` already exists, so it is more appropriate to replace it with the following code. WYT 🤔
```js
  <div v-if="$slots.loading && loading" :class="nsSelect.be('dropdown', 'loading')">
      <slot name="loading" />
   </div>
```

Reply:
Well, the `<template #loading />` is for `el-select-menu`, and the `<slot name="loading" />` is for `el-select-v2`. It can not be removed.